<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="3" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_37_IST_2019.jpg>Screenshot link</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_37_IST_2019.jpg><img height=200 width=200 src=Sun_Apr_21_11_07_37_IST_2019.jpg></a>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_41_IST_2019.jpg>Screenshot link</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_41_IST_2019.jpg><img height=200 width=200 src=Sun_Apr_21_11_07_41_IST_2019.jpg></a>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_44_IST_2019.jpg>Screenshot link</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_44_IST_2019.jpg><img height=200 width=200 src=Sun_Apr_21_11_07_44_IST_2019.jpg></a>]]>
    </line>
  </reporter-output>
  <suite name="TestNG Learning" duration-ms="7905" started-at="2019-04-21T05:37:37Z" finished-at="2019-04-21T05:37:45Z">
    <groups>
    </groups>
    <test name="Bank Manager Login Test" duration-ms="912" started-at="2019-04-21T05:37:37Z" finished-at="2019-04-21T05:37:38Z">
      <class name="testcases.BankManagerLoginTest">
        <test-method status="FAIL" signature="loginAsBankManager()[pri:0, instance:testcases.BankManagerLoginTest@68999068]" name="loginAsBankManager" duration-ms="453" started-at="2019-04-21T05:37:37Z" finished-at="2019-04-21T05:37:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Bank Manager Login Test Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Bank Manager Login Test Failed
at org.testng.Assert.fail(Assert.java:96)
at testcases.BankManagerLoginTest.loginAsBankManager(BankManagerLoginTest.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_37_IST_2019.jpg>Screenshot link</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_37_IST_2019.jpg><img height=200 width=200 src=Sun_Apr_21_11_07_37_IST_2019.jpg></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginAsBankManager -->
      </class> <!-- testcases.BankManagerLoginTest -->
    </test> <!-- Bank Manager Login Test -->
    <test name="Add Customer Test" duration-ms="3612" started-at="2019-04-21T05:37:38Z" finished-at="2019-04-21T05:37:41Z">
      <class name="testcases.AddCustomerTest">
        <test-method status="FAIL" signature="addCustomer(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.AddCustomerTest@dbf57b3]" name="addCustomer" duration-ms="3088" started-at="2019-04-21T05:37:38Z" data-provider="getData" finished-at="2019-04-21T05:37:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Raman]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Arora]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[A234324]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Add Customer Test Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Add Customer Test Failed
at org.testng.Assert.fail(Assert.java:96)
at testcases.AddCustomerTest.addCustomer(AddCustomerTest.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_41_IST_2019.jpg>Screenshot link</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_41_IST_2019.jpg><img height=200 width=200 src=Sun_Apr_21_11_07_41_IST_2019.jpg></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- addCustomer -->
      </class> <!-- testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
    <test name="Open Account Test" duration-ms="3339" started-at="2019-04-21T05:37:42Z" finished-at="2019-04-21T05:37:45Z">
      <class name="testcases.OpenAccountTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.OpenAccountTest@42e99e4a]" name="setUp" is-config="true" duration-ms="38708" started-at="2019-04-21T05:36:58Z" finished-at="2019-04-21T05:37:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="openAccount(java.lang.String, java.lang.String)[pri:0, instance:testcases.OpenAccountTest@42e99e4a]" name="openAccount" duration-ms="2895" started-at="2019-04-21T05:37:42Z" data-provider="getData" finished-at="2019-04-21T05:37:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Raman Arora]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rupee]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Open Account Test Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Open Account Test Failed
at org.testng.Assert.fail(Assert.java:96)
at testcases.OpenAccountTest.openAccount(OpenAccountTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_44_IST_2019.jpg>Screenshot link</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Sun_Apr_21_11_07_44_IST_2019.jpg><img height=200 width=200 src=Sun_Apr_21_11_07_44_IST_2019.jpg></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- openAccount -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.OpenAccountTest@42e99e4a]" name="tearDown" is-config="true" duration-ms="1135" started-at="2019-04-21T05:37:45Z" finished-at="2019-04-21T05:37:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.OpenAccountTest -->
    </test> <!-- Open Account Test -->
  </suite> <!-- TestNG Learning -->
</testng-results>
